airflow/config_templates/default_airflow.cfg

```diff
@@ -325,6 +325,11 @@ scheduler_zombie_task_threshold = 300
 # DAG definition (catchup)
 catchup_by_default = True
 
+# This changes the batch size of queries in the scheduling main loop.
+# This depends on query length limits and how long you are willing to hold locks.
+# 0 for no limit
+max_tis_per_query = 0
```

Comment:
I think I'd rather have something more verbose here if it makes it clearer. I think expecting users to find the default airflow config to get an idea of what this does isn't great. I guess it's a bit of a nit, but anything that's exposed to users should probably have a bit of care put in to it.